tests/BladeTest.php

```diff
@@ -117,7 +117,7 @@ public function the_hasanyrole_directive_will_evaluate_false_when_the_logged_in_
 
         auth()->setUser($this->getMember());
 
-        $this->assertEquals('does not have any of the given roles', $this->renderView('hasAnyRole', compact('roles')));
+        $this->assertEquals('does not have any of the given roles', $this->renderView('hasAnyRole', ['roles' => implode('|', $roles)]));
```

Comment:
Could you, instead of replace the current assertions, create a PR that adds your news ones. You can put them in the same methods as the current assertions.